obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  mp_starcat_time: 2011:094:17:16:49.408
+  status: 1
+- agasc_id:
+  - 647762704
+  comments: Almost never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2011:108:03:39:32.321
+  status: 1
+- agasc_id:
+  - 648676152
+  comments: High Background Event
+  mp_starcat_time: 2011:122:08:00:41.258
+  status: 1
+- agasc_id:
+  - 656409216
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2011:152:21:37:04.174
+  status: 1
+- agasc_id:
+  - 665715168
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741605832
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741605976
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741607368
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741609000
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741610072
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741736560
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 741736840
+  comments: Safe mode
+  mp_starcat_time: 2011:187:11:50:08.646
+  status: 1
+- agasc_id:
+  - 225576592
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2011:248:04:00:06.000
+  status: 1
+- agasc_id:
+  - 614482704
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2011:269:04:58:50.064
+  status: 1
+- agasc_id:
+  - 83631240
+  comments: High background event
+  mp_starcat_time: 2011:275:21:41:33.299
+  status: 1
+- agasc_id:
+  - 764150080
+  comments: Never acquired; set mag 12.0 +/- 0.1
+  mp_starcat_time: 2011:285:11:34:00.061
+  status: 1
+- agasc_id:
+  - 766774840
+  comments: Never acquired
+  mp_starcat_time: 2011:287:18:23:11.490
+  status: 1
+- agasc_id:
+  - 618535032
+  comments: Almost never acquired
+  mp_starcat_time: 2011:309:05:29:06.000
+  status: 1
+- agasc_id:
+  - 306200176
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2011:338:07:22:12.671
+  status: 1
+- agasc_id:
+  - 1058938112
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2011:343:07:05:57.421
+  status: 1
+- agasc_id:
+  - 1092619768
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2012:028:20:14:46.841
+  status: 1
+- agasc_id:
+  - 40113272
+  comments: Faint star. Affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2012:065:14:30:29.432
+  status: 1
+- agasc_id:
+  - 118359864
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2012:092:01:34:23.764
+  status: 1
+- agasc_id:
+  - 664279632
+  comments: Almost never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2012:174:04:58:32.901
+  status: 1
+- agasc_id:
+  - 653529952
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2012:187:04:30:17.984
+  status: 1
+- agasc_id:
+  - 740693864
```

Comment:
Not sure if obsid 54511 should be added to bad star-obs too (it doesn't look much better than 54506 here).  But probably no effective difference if star added to bad list and mags.